libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -207,31 +262,34 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   @HostListener('window:focusout')
   onFocusChange() {
     // This fixes an undesired scroll behaviour occurring on keyboard-tabbing backwards (with shift+tab):
-    window.scrollTo({ top: this.scrollY });
+    this.windowRef.scrollTo({ top: this.scrollY });
   }
 
-  @HostListener('window:keyboardWillShow', ['$event'])
-  _onKeyboardWillShow(info?: { keyboardHeight: number }) {
+  @HostListener('window:ionKeyboardDidShow', ['$event'])
+  _onKeyboardWillShow(event: { detail: { keyboardHeight: number } }) {
     this.keyboardVisible = true;
-    if (info && info.keyboardHeight) {
-      this.setKeyboardOffset(info.keyboardHeight);
-    }
+    this.setKeyboardOffset(event.detail.keyboardHeight);
+    const ionModalWrapper = this.elementRef.nativeElement.closest<HTMLElement>('.modal-wrapper');
+    if (!ionModalWrapper) return;
+    const distanceFromWindowBottomToModalBottom =
+      this.windowRef.innerHeight - ionModalWrapper.getBoundingClientRect().bottom;
+    const keyboardOverlap = this.keyboardHeight - distanceFromWindowBottomToModalBottom;
+    console.warn('keyboardOverlap:', keyboardOverlap);
```

Comment:
A little too much copy paste from Slack 😂
Delete console.warn, and use `keyboardOverlap` in `this.setKeyboardOffset`... 
```suggestion
    const ionModalWrapper = this.elementRef.nativeElement.closest<HTMLElement>('.modal-wrapper');
    if (!ionModalWrapper) return;
    const distanceFromWindowBottomToModalBottom =
      this.windowRef.innerHeight - ionModalWrapper.getBoundingClientRect().bottom;
    const keyboardOverlap = event.detail.keyboardHeight - distanceFromWindowBottomToModalBottom;
        this.setKeyboardOffset(keyboardOverlap);
```